modes/documentation-mode/internal.lisp

```diff
@@ -64,6 +67,7 @@
             (enough-namestring (location-file location) (asdf:system-source-directory :lem))
             (location-line-number location))))
 
+#+sbcl
```

Comment:
Since separation by `*features*` is complicated, I think it is preferable to keep it to a minimum.
except sbcl, what about command-definition-location returning meaningless values (e.g., always set position to 0)?

Reply:
You mean something like this https://github.com/lem-project/lem/pull/756/commits/52a90ebb3c0cd6e99330a6a6c116113e9b2d1789?

Reply:
Yes, I thought it would be better to consolidate on the called side rather than the caller.
How about setting the file name to nil to indicate that it has no meaning?

Reply:
Done :+1: